rllib/agents/ppo/ppo_torch_policy.py

```diff
@@ -119,7 +117,10 @@ def ppo_surrogate_loss(policy, model, dist_class, train_batch):
     mask = None
     if state:
         max_seq_len = torch.max(train_batch["seq_lens"])
-        mask = sequence_mask(train_batch["seq_lens"], max_seq_len)
+        mask = sequence_mask(
+            train_batch["seq_lens"],
+            max_seq_len,
+            time_major=model.model_config.get("_time_major", False))
```

Comment:
Should this be something like `model.is_time_major()`?

Reply:
great idea.